rules/windows/evasion_suspicious_wmi_script.toml

```diff
@@ -0,0 +1,48 @@
+[metadata]
+creation_date = "2020/09/02"
+ecs_version = ["1.6.0"]
+maturity = "development"
```

Comment:
just confirming this is not meant for prod

Reply:
This _was_ intentional, unless we think the wildcard performance concerns are negligible

Reply:
I'll bump it and see how it works out

```suggestion
maturity = "production"
```